### ISSUE TITLE
Error when running python3 generate_voice_pack.py

### ISSUE THREAD

**Issue by killmanda2** [cktlco/crew-chief-autovoicepack#34]
Hello.
Thanks for your hard work.
Please help me.
When I run python3 generate_voice_pack.p I get two errors:

_class "__nv_bfloat162" has already been defined (previous definition at line 4787 of /usr/local/cuda-12.6/include/cuda_bf16.h)_.

All installations were carried out according to the instructions. Installation on Windows 11.

![Screenshot 2024-11-28 111803](https://github.com/user-attachments/assets/67da4921-8d15-4192-ad80-ebeacbf4b7dc)

**Comment by cktlco** [comment]
Hi, thanks for reaching out, we'll get it working for you. Those messages look like the normal and very verbose warnings/errors from the Deepspeed library.

The process continues after that message, right? I expect messages like that (I can't really suppress them since it's in third-party code), but it should continue past that section after a few minutes of initializing.

Does the entire process stop when you see that error? You can also try `--disable_deepspeed` to perhaps bypass that section entirely.

**Comment by cktlco** [comment]
Hi, I tried to reproduce, but on a machine with an Nvidia RTX 3090 I did not see the same errors you show above. If you are trying to run with an Nvidia GPU, maybe its a model not supported by deepspeed, in which case maybe the process will just work for you using `--disable_deepspeed` as part of `python3 generate_voice_pack.py ...`.

Additionally, everything will still work using by using `--cpu_only` mode, just running more slowly than if you are using a modern nvidia GPU. Maybe try that as a fallback.
